app/views/videos/show.html.haml

```diff
@@ -7,4 +7,36 @@
         %header
           %h3= @video.title
           %span Rating: 4.5/5.0
-        %p= @video.description
\ No newline at end of file
+        %p= @video.description
+        .actions
+          = link_to "+ My Queue", queue_items_path(video_id: @video.id), method: :post, class: "btn btn-default"
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            = f.label "Rate this video"
+            .row
+              .col-sm-3
+                = f.select :rating, options_for_select([5,4,3,2,1].map { |number| [pluralize(number, "Star")] }), {}, { class: "form-control"}
```

Comment:
The result array should also contain the number itself - like ["5 Starts", 5] etc. Because you want to pass only the ID to the controller, not the id with some text. Try to figure out how to fix this, also a helper method would be better :)